src/tests/system/tests/test_failover.py

```diff
@@ -106,3 +106,37 @@ def test_failover__connect_using_ipv4_second_family(client: Client, provider: Ge
 
     result = client.tools.id(user.name)
     assert result is not None, f"{user.name} was not found, SSSD did not switch to IPv4 family!"
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.IPA)
+@pytest.mark.topology(KnownTopology.LDAP)
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.require(
+    lambda client, sssd_service_user: (client.features["ipv6"]),
+    "SSSD was built without support for running under non-root",
+)
+def test_failover__connect_using_ipv6_first(client: Client, provider: GenericProvider, sssd_service_user: str):
+    """
+    :title: Only attempt to resolve hostnames to IPv6 address
+    :setup:
+        1. Create user
+        2. Set family_order to "ipv6_only"
+        3. Set IPv6 address in /etc/hosts
```

Comment:
Now you are setting IPv4 address. It would be good to mention that this address is ureachable (Set unreachable IPv4 address...)